source/_integrations/dublin_bus_transport.markdown

```diff
@@ -10,6 +10,9 @@ ha_platforms:
   - sensor
 ha_integration_type: integration
 ---
+<div class='note warning'>
+The API used at [Dub Linked](https://data.smartdublin.ie/) is no longer accessible, rendering this integration unusable.
+</div>
```

Comment:
We don't track issues in our documentation.

As there is an issue open in our Core issue tracker, please watch that one instead.

../Frenck